package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ual-plainjs-renderer",
-  "version": "0.1.6",
```

Comment:
If you only updated example code, I don't think is actually needed. Since we don't publish the example in the npm package, "nothing" has actually changed.

Reply:
but to publish to npm repo, the parent version needs to change, yes?